content/ReleaseNotes/TrueCommand/2.0-Beta.md

```diff
@@ -0,0 +1,115 @@
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1626'>TC-1626</a>] -         Add navigation shortcut to Cluster Manager
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1643'>TC-1643</a>] -         NAS Gluster APIs Update
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1649'>TC-1649</a>] -         IP mapping fixes and other data flow issues
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1656'>TC-1656</a>] -         Config Backup Dialog Box has no tooltips for icons
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1710'>TC-1710</a>] -         Cleanup Dashboard card: Rework network/disk meters.
+</li>
+</ul>
+
+
+
+### Bug Fixes
+
+<ul>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1602'>TC-1602</a>] -         Can't connect to TrueNAS-12.0-U2.1 with HTTP to HTTPS redirect enabled
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1640'>TC-1640</a>] -         Incorrect TrueCommand Email Alert Subject Lines
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1650'>TC-1650</a>] -         Save/Cancel buttons in configuration dialog not active
+</li>
+<li>[<a href='https://jira.ixsystems.com/browse/TC-1670'>TC-1670</a>] -         Description is not saved when creating/editing a Report
+</li>
+</ul>
+
+### Known Issues
+
+<body class="ql-editor ql-editor-view" style="font-size:14px;">
+    <html>
+        <body>
+            <table width="100%">
+                <thead><tr><th>Key</th><th>Summary</th><th>Workaround</th></tr></thead>
+                <tbody>
+                   <tr>
+                      <td><a href="https://jira.ixsystems.com/browse/NAS-110413" target="_blank">NAS-110413</a></td>
+                      <td>The UI is misreading the schedule for a Snapshot Task from True Command.</td>
+                      <td></td>
+                  </tr>
+                  <tr>
+                      <td></td>
+                      <td>Cluster Volumes.</td>
+                      <td> This is a brand new feature and is still being polished.</td>
+                  </tr>
+                      <td></td>
+                      <td>iSCSI Management.</td>
+                      <td> This is a brand new feature and is still being polished.</td>
+                  </tr>
+			         </tbody>
+            </table>
+        </body>
+    </html>
+
+## To Download this Release
+ 
+Login to the [TrueCommand Account Portal](https://portal.ixsystems.com) for downloads, documentation links, and licensing options.
+For storage clusters with more than 50 disks, the account portal also offers a *free 60-day trial license with unlimited disks*.
+ 
+## To Update to this Release
+ 
+**Docker:** Re-run `docker pull ixsystems/truecommand:2.0_beta` to fetch this version of TrueCommand, and then restart your docker instance.
+ 
+{{< hint warning >}}
+**Pre-Release Testing Version**
+This is a BETA release and is not usable for production systems.
```

Comment:
"not recommended for production systems"

Reply:
Changed